src/annotator/components/toolbar.js

```diff
@@ -0,0 +1,135 @@
+import classnames from 'classnames';
+import propTypes from 'prop-types';
+import { createElement } from 'preact';
+
+function ToolbarButton({
+  buttonRef,
+  extraClasses,
+  label,
+  icon,
+  onClick,
+  selected,
+}) {
+  const handleClick = event => {
+    event.stopPropagation();
+    onClick();
+  };
+
+  return (
+    <button
+      // This currently uses the icon font. This needs to be converted to
+      // `SvgIcon`.
```

Comment:
Are we prevented from doing this (now) by the absence of these icons in SVG format, or by some other technical impediment?

Reply:
We don't have all these icons in SVG format already in the repo, so deferred this to follow-up work.